app/helpers/blacklight_helper.rb

```diff
@@ -10,9 +10,11 @@ def document_partial_name(document, base_name = nil)
 
     display_type ||= document.display_type
 
+    display_type ||= document['eds_publication_type']
```

Comment:
It's set to `format` in the show config -- see https://github.com/sul-dlss/SearchWorks/blob/article-search/app/controllers/article_controller.rb#L84